tendermint/src/amino_types/proposal.rs

```diff
@@ -172,7 +173,7 @@ impl SignableMsg for SignProposalRequest {
 }
 
 impl ConsensusMessage for Proposal {
-    fn validate_basic(&self) -> Result<(), ValidationError> {
+    fn validate_basic(&self) -> Result<(), Error> {
```

Comment:
As there are multiple error types used in this module, what about qualifying all of them. This one `errror::Error` and `validate::Error` for the others.